packages/v4/src/content/training/trainingCard/trainingCard.js

```diff
@@ -50,10 +51,10 @@ export const TrainingCard = ({
       {description}
     </CardBody>
     <CardFooter>
-    <Link to={`https://www.katacoda.com/patternfly/courses/${trainingType}/${katacodaId}`} target="_blank">
+    <Link to={`/training/${subsection}/${katacodaId}`} >
```

Comment:
I think you should just align `trainingType` and `subsection`. Just change `subsection: react` to `subsection: React charts` for the charts examples.